terraform/aio/scripts/configure-local-networking.sh

```diff
@@ -9,50 +9,50 @@ cat << EOF | sudo tee -a /etc/hosts
 EOF
```

Comment:
Apart from this part, this script is a noop. Will be interesting to see how CentOS copes with that.

Reply:
Actually we can drop this due to the userdata.cfg.tpl change

Reply:
Oh, actually we've' dropped this file from the terraform config entirely.